### DISCUSSION TITLE
Why is the Authentication link broken on the Request Standards | Etsy Open API v3 page?

### DISCUSSION THREAD

**Discussion by edmiidz** [etsy/open-api#852] [Q&A]
Why is the [Authentication link](https://developers.etsy.com/documentation/essentials/requests/authentication) broken on the [Request Standards | Etsy Open API v3 page](https://developers.etsy.com/documentation/essentials/requests/)?

I get a 404 not found when I click on it.

**Comment by PMStanley** [comment]
Where are you seeing that url?  The Authentication link on the Request Standards page is https://developers.etsy.com/documentation/essentials/authentication which is working.

Oh I see, following your link gets you a version of the page which has the wrong Auth link in the header table.  If you click on any of the sections in the left nav bar and then click Request Standards you get a working link in the table.

It's the trailing / in your Request Standards URL.  The website doesn't have one and it's presence in yours is causing the Auth link to map to the wrong level

**Comment by edmiidz** [comment]
About half way down the [Request Standards | Etsy Open API v3 page](https://developers.etsy.com/documentation/essentials/requests/).
![image](https://user-images.githubusercontent.com/610475/217083854-5e3aeac0-04db-4a6a-b7a6-f2254d990428.png)

**Reply by PMStanley** [reply]
Yes, did you see that I edited my reply to explain why it doesn't work for you?
